### DISCUSSION TITLE
Increasing the computational speed along with accuracy in Xcompact3D

### DISCUSSION THREAD

**Discussion by gauravity30** [xcompact3d/Incompact3d#277] [Q&A] [answered]
Hello,
I recently tried Nek5000, which is quite fast but has issues with creating mesh for complex geometries/non-canonical flows on the other hand xcompact3d doesn't have such issues. I am aiming to perform DNS/LES for complex immersed bodies for moderate Re range 15-25K. Can you suggest some methods to increase the computational speed while maintaining good accuracy ? 

The geometry has small/thin parts which may require high number of grid points in all directions.  

Thanks,
Gaurav.

**Comment by slaizet** [comment, marked as answer]
Hi Gaurav,
I am not quite sure to understand the purpose of this message. You cannot magically increase the computational speed of Xcompact3d while maintaining the sixth-order accuracy. If you want to reduce your time to solution, then you can increase the number of MPI processes and/or run your simulation on recent hardware (both interconnect and CPUs).
Regarding local refinement, this is not an option in Xcompact3d so simulations at high Reynolds numbers around a bluff body are not a good fit for Xcompact3d, except if you have a wall model. That is why we do not simulate flow around airfoils or turbomachinery applications.
Best,
Sylvain
